File: src/main/java/com/team04/back/global/config/SpringDocConfig.java
package com.team04.back.global.config;

import io.swagger.v3.oas.annotations.OpenAPIDefinition;
import org.springframework.context.annotation.Configuration;

@Configuration
@OpenAPIDefinition(
        info = @io.swagger.v3.oas.annotations.info.Info(
                title = "API documentation",
                version = "v1",
                description = "API documentation for Team04 project"
        )
)
public class SpringDocConfig {
}

File: src/main/java/com/team04/back/global/config/SpringDocConfig.kt
package com.team04.back.global.config

import io.swagger.v3.oas.annotations.OpenAPIDefinition
import io.swagger.v3.oas.annotations.info.Info
import org.springframework.context.annotation.Configuration

@Configuration
@OpenAPIDefinition(
    info = Info(
        title = "API documentation",
        version = "v1",
        description = "API documentation for Team04 project"
    )
)
class SpringDocConfig 
